js/bitfinex.js

```diff
@@ -671,6 +671,29 @@ module.exports = class bitfinex extends Exchange {
             'AVT': 'aventus', // #1811
             'QTUM': 'qtum', // #1811
             'EDO': 'eidoo', // #1811
+
```

Comment:
This empty line should be deleted, as it breaks the transpiler rule from here:
https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

Apart from this empty line, the rest of the code is clean. Thx!